Example/FusumaExample.xcodeproj/project.pbxproj

```diff
@@ -305,6 +305,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				INFOPLIST_FILE = FusumaExample/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				ONLY_ACTIVE_ARCH = NO;
 				PRODUCT_BUNDLE_IDENTIFIER = me.ytakzk.FusumaExample;
```

Comment:
No we don't. I have the habit of changing this flag to No while compiling. Sorry for that.